db/migrate/20160417105237_add_read_only_to_policies.rb

```diff
@@ -0,0 +1,6 @@
+class AddReadOnlyToPolicies < ActiveRecord::Migration[5.0]
+  def change
+    add_column :miq_policies, :read_only, :boolean, :default => false
+    add_column :conditions,   :read_only, :boolean, :default => false
```

Comment:
Please remove the default values...nil is considered falsey, and is the default already